otter/integration/tests/test_scaling.py

```diff
@@ -123,3 +119,63 @@ def test_scaling_up(self):
         )
         return d
     test_scaling_up.timeout = 1800
+
+    def test_policy_execution_after_adding_clb(self):
```

Comment:
This is WIP, so no docstring yet.  What form should it take?  A link to the bug?